tests/integration/test_user_grants_from_config/configs/another_user.xml

```diff
@@ -0,0 +1,18 @@
+<clickhouse>
+    <users>
+        <another>
+            <password/>
+            <networks>
+                <ip>::/0</ip>
+            </networks>
+            <profile>default</profile>
+            <quota>default</quota>
+            <grants>
+                <query>GRANT SHOW ON *.*</query>
+                <query>GRANT CREATE ON *.* WITH GRANT OPTION</query>
+                <query>GRANT SELECT ON system.*</query>
+                <query>REVOKE CREATE ON system.*</query>
```

Comment:
Could you please add one example with XML escaping?

Reply:
I don't quite understand what I can possibly escape in those queries. Can you give me a hint?